lib/backend/sanitize.go

```diff
@@ -16,29 +15,11 @@ const errorMessage = "special characters are not allowed in resource names, plea
 
 // whitelistPattern is the pattern of allowed characters for each key within
 // the path.
-var whitelistPattern = regexp.MustCompile(`^[0-9A-Za-z@_:.-]*$`)
+var whitelistPattern = regexp.MustCompile(`^[0-9A-Za-z@_:.\-/]*$`)
```

Comment:
Did you mean to add forward and backward slashes to the whitelist? The way you added them with the dash in-between - doesn't it make it a range i.e. from `\` char to `/` char in ASCII table (which wouldn't make sense because `\` appears later in the table).

Reply:
I meant to add `-` and forward `/`, so looks ok

Reply:
in `isStringSafe()` below it actually looks for and blocks the path separator explicitly, is adding a slash here safe?

Reply:
it did before, and it's not relevant any more, as directory based backend is gone and is replaced by sqlite that does not care about any local path problems